packages/icons/component-template.ts

```diff
@@ -0,0 +1,60 @@
+// @ts-check
+
+import { IconData } from './processIcons';
+
+export const getComponentName = (name: string): string => {
+  const suffix = name === 'Props' ? '_Props' : name;
+  return `Icon${suffix}`;
```

Comment:
I'd added the `Icon` prefix as a naming convention since it makes it easier to tell that the components are simple icon components rather than anything more than that. It's been pretty standard practice in the design systems I've worked on, but we can change it if you'd rather it be the other way :)

Reply:
I think it would be nicer because we are already importing each component from a lib that only has its purpose and the user is able to rename it if they want. I also think it is cleaner.

```javascript
import { WebOpen as IconWebOpen, NotePad as IconNotePad} from '@react95/icons';
```

Reply:
I think it would be nicer because we are already importing each component from a lib that only has only the purpose to export Icons and the user is able to rename it if they want. I also think it is cleaner.

```javascript
import { WebOpen as IconWebOpen, NotePad as IconNotePad} from '@react95/icons';
``` 

After a little research: 
 - [Ant Icons](https://ant.design/components/icon/): without prefix
 - [Material Icons](https://material-ui.com/components/material-icons/): without prefix
 - [React Icons](https://react-icons.github.io/react-icons): with prefix (and I think the prefix is to distinguish the vendors)

Reply:
Cool, I'll update it to be that. I think it's a bit clear away from the import statement, especially since we'll end up with some components with names like `Props`. But like you said, people can always rename them

Reply:
Hey, @kirkobyte 

I've just done that in my last commit. Could you check it?

I think we are done with this implementation and ready to merge it!

Reply:
Cool, looks good to go to me! :)